test/cpp_extensions/open_registration_extension/torch_openreg/torch_openreg/_utils.py

```diff
@@ -0,0 +1,86 @@
+import ctypes
+import glob
+import os
+import platform
+import sys
+import sysconfig
+import textwrap
+
+
+def _load_dll_libraries() -> None:
```

Comment:
This is surprising that we have to do so much here? We don't usually do that for our cpp extensions.